packages/camera/src/components/UploadCenter/index.js

```diff
@@ -56,8 +59,41 @@ export default function UploadCenter({
     return indexB - indexA;
   }, [sights.state.tour]);
 
+  const fulfilledUploads = useMemo(() => Object.values(uploads.state)
+    .filter(({ status }) => status === 'fulfilled'), [uploads.state]);
+
   const fulfilledCompliance = useMemo(() => Object.values(compliance.state)
-    .filter(({ status }) => status === 'fulfilled'), [compliance.state]);
+    .filter(({ status }) => status === 'fulfilled')
+    .map(({ result, ...item }) => {
+      const carCov = result.data.compliances.coverage_360;
+      const iqa = result.data.compliances.image_quality_assessment;
+
+      // `handleChangeReasons` returns the full result object with the given compliances
+      const handleChangeReasons = (compliances) => ({
+        ...item,
+        result: { ...result,
+          data: { compliances: { ...result.data.compliances, ...compliances } } } });
+
+      // if status is TODO, mark it as compliant (ignore)
```

Comment:
more comment is needed here to say that this is a kludge and we need to change it